express-get-json/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesData = [];
+for (const property in grades) {
+  gradesData.push(grades[property]);
+}
+
+app.get('/api/grades', function (req, res) {
+  // console.log(req.method);
+  res.json(gradesData);
+});
```

Comment:
Try building your array inside of your route callback.

Reply:
![Screen Shot 2021-12-02 at 12 23 23 AM](https://user-images.githubusercontent.com/72685552/144384704-20ce4a4d-9ca3-4f33-98da-4edd37c194d7.png)